README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
```

Comment:
I appreciate the use of the list function, I think it helps exemplify all the places that you have lived. I am sure that the weather here is much different than the places you have lived previously. I hope you are not too cold!